pkg/schedule/schedulers/evict_slow_store.go

```diff
@@ -162,14 +151,16 @@ func (handler *evictSlowStoreHandler) updateConfig(w http.ResponseWriter, r *htt
 		handler.rd.JSON(w, http.StatusInternalServerError, errors.New("invalid argument for 'recovery-duration'").Error())
 		return
 	}
-	handler.config.Lock()
-	defer handler.config.Unlock()
-	prevRecoveryDurationGap := handler.config.RecoveryDurationGap
+
+	conf := handler.config
```

Comment:
Is the `conf` assignment also need to be in the lock scope?

Reply:
This is a pointer, so this assignment won't trigger any data reading or writing.

But to avoid similar misunderstandings, I'll revert this change, it won't affect this pull request.